src/smokeTest/java/uk/gov/hmcts/reform/bulkscanprocessor/TestHelper.java

```diff
@@ -133,13 +137,12 @@ public CloudBlobContainer getCloudContainer(
         return new CloudBlobContainer(PathUtility.addToQuery(containerUri, sasToken));
     }
 
-    public String getRandomFilename(String suffix) {
-        StringBuilder strBuffer = new StringBuilder();
-        strBuffer
-            .append(ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE))
-            .append(Strings.isNullOrEmpty(suffix) ? "" : "_")
-            .append(Strings.isNullOrEmpty(suffix) ? "" : suffix);
-        return strBuffer.toString();
+    public String getRandomFilename() {
```

Comment:
current one prefix random number to the given file name which should also result in unique 🤔

Reply:
Ithe filename used to be generated in this format:  `ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE)` + `_<<suffix>>`
It looks like, `ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE)` is returning duplicates

Reply:
discussed on slack looks like `ThreadLocalRandom.current().nextInt` is generating same number

Reply:
yes it will return. random is never truly random. should not blindly trust it